tests/run_tests.py

```diff
@@ -351,28 +357,37 @@ def run_system(config_file, vic_exe, test_data_dir, out_dir, driver):
                                                         expectation))
 
             # -------------------------------------------------------- #
-            # check output files
-            if test_dict['check']:
-                if 'complete' in test_dict['check']:
-                    start = None
-                    end = None
-
-                for fname in glob.glob(os.path.join(dirs['results'], '*')):
-                    df = read_vic_ascii(fname, header=True)
-                    # do numeric tests
-
-                    # check that each dataframe includes all timestamps
+            # check output files (different tests depending on driver)
+            if driver == "classic":
```

Comment:
I like this idea. I've standardized the test name to 'output_nans' for both drivers.